src/Calculator/Controls/EquationTextBox.cpp

```diff
@@ -323,3 +352,11 @@ bool EquationTextBox::ShouldDeleteButtonBeVisible()
     }
     return (!text->IsEmpty() && m_HasFocus);
 }
+
+void EquationTextBox::OnRichEditMenuOpening(Platform::Object ^ sender, Platform::Object ^ args)
+{
+    if (m_kgfEquationMenuItem != nullptr)
+    {
+        m_kgfEquationMenuItem->IsEnabled = EquationTextBox::RichEditHasContent();
```

Comment:
KGF button should also be disabled/collapsed when the equation is in an error state but would still have richedit content.

Reply:
Unfortunately, we don't know if it's in an error state or not because we'd have to calculate it on keystroke, which would be heavy.  This is something of a compromise where we will enable it whenever there is a chance that it is a valid equation.